action_files/test_models/src/multivariate_models.py

```diff
@@ -0,0 +1,53 @@
+import os
+import time
+
+import fire
+import pandas as pd
+
+from neuralforecast.core import NeuralForecast
+
+from neuralforecast.models.softs import SOFTS
+from neuralforecast.models.tsmixer import TSMixer
+from neuralforecast.models.tsmixerx import TSMixerx
+from neuralforecast.models.itransformer import iTransformer
+from neuralforecast.models.stemgnn import StemGNN
+from neuralforecast.models.mlpmultivariate import MLPMultivariate
+
+from neuralforecast.losses.pytorch import MAE
+
+from src.data import get_data
+
+os.environ['NIXTLA_ID_AS_COL'] = '1'
+
+
+def main(dataset: str = 'multivariate', group: str = 'ETTm2') -> None:
+    train, horizon, freq = get_data('data/', dataset, group)
+    train['ds'] = pd.to_datetime(train['ds'])
+
+    models = [
```

Comment:
I'd add all multivariate models here

Reply:
Same here, not sure which model I'm missing. We have SOFTS, TSMixer/TSMixerx, iTransformer, MLPmultivariate, StemGNN. Missing TimeMixer, but that PR is not merged yet.